lib/index.js

```diff
@@ -1,8 +1,13 @@
 'use babel';
 
+import * as fs from 'fs';
```

Comment:
If possible these modules should be lazily loaded, I've used [lazy-req](https://www.npmjs.com/package/lazy-req) in the past, but if you have a better method go for it :wink:.

Right now this will increase the package loading time as these will be required on Atom launch.